start.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Dennis Richie was brilliant and was right 80%+ of the time (which is one of the things that has made him a legend).

This was a case that fell into the 20%-. The invention of interpreter directives in Unix was not a good approach, and in Linux, production environments (should) turn off the `binfmt_misc` filesystem.

A vastly more portable approach, rather than setting execute bit and using an interpreter directive, is to directly specify the interpreter.

```
bash ./start.sh
```

This forces a few good habits, such as specifying the absolute path of a binary and specification of the interpreter to use with a script.